src/librawspeed/decoders/RawDecoder.cpp

```diff
@@ -125,8 +125,6 @@ void RawDecoder::decodeUncompressed(TiffIFD *rawIFD, BitOrder order) {
 
 bool RawDecoder::checkCameraSupported(CameraMetaData *meta, string make,
                                       string model, const string &mode) {
-  TrimSpaces(make);
```

Comment:
But it is not guaranteed that the function is called with already-trimmed values?
E.g. see aridecoder, it calls with some string